assets/ops/config.fish

```diff
@@ -15,6 +15,7 @@ alias ml "mix lint"
 alias mer "mix ecto.reset"
 
 function setup
+    /setup_security.sh
```

Comment:
true, this script copies the certs into a directory of the container and destroyed after the container is destroyed. We could add a local mapping that will allow them to be more permanent, which is what we do in other projects.

Reply:
Oh, yeah, in that case, we might be ok since the changes the script makes do already persist between container restarts. I might have been thinking that it sets environment variables.